helpers/deploymentConfig.ts

```diff
@@ -1759,6 +1777,12 @@ export const globalConfig: NetworkConfig = {
             supplySpeeds: ["3703703703703703"], // 3200 SD over 30 days (864000 blocks)
             borrowSpeeds: ["3703703703703703"], // 3200 SD over 30 days (864000 blocks)
           },
+          {
+            asset: "SD",
+            markets: ["BNBx"],
+            supplySpeeds: ["1157407407407407"], // 1000 SD over 28 days (806400 blocks)
+            borrowSpeeds: ["1157407407407407"], // 1000 SD over 28 days (806400 blocks)
```

Comment:
```suggestion
            supplySpeeds: ["1240079365079365"], // 1000 SD over 28 days (806400 blocks)
            borrowSpeeds: ["1240079365079365"], // 1000 SD over 28 days (806400 blocks)
```

Reply:
The value is correct, the comment is wrong. I'll fix it